lib/logux.rb

```diff
@@ -78,7 +78,7 @@ def self.add_batch(commands)
   def self.undo(meta, reason: nil, data: {})
     add(
       data.merge(type: 'logux/undo', id: meta.id, reason: reason),
-      Logux::Meta.new(clients: [meta.client_id])
+      meta.undo_meta
```

Comment:
I have an idea to include `action` to `logux/undo`at some moment in the future. But I think we can do it in this way right now and refactor it later.

Reply:
Thank you for the review! I'll finish the edits here after current logux_rails/logux-rack updates are integrated.